src/workflows/registration.ts

```diff
@@ -54,14 +42,25 @@ export async function registerOffchainWorkflow({
     },
   });
 
-  return;
+  return registeredUser.data;
```

Comment:
We decided that this should be fixed in the upstream api-level as it always returns `{"tx_hash": ""}`. Have we followed up / made a ticket to action this? If not, would be good as we will likely not revisit this for a while.

Reply:
Created DX-1265 for followup.